10-inference.Rmd

```diff
@@ -54,15 +58,15 @@ In the above question, we are interested in making a conclusion about *all*
 undergraduate students in North America; this is our **population**.
 In general, the population is the complete collection of individuals or cases we are interested in studying. 
 Further, in the above question, we are interested in computing a quantity&mdash;the proportion
-of iPhone owners&mdash;based on the entire population. This is our **population parameter**.
+of iPhone owners&mdash;based on the entire population. This proportion is our **population parameter**.
 In general, a population parameter is a numerical characteristic
 of the entire population. To compute this number in the example above, we would need to ask 
-every single undergraduate in North America whether or not they own an iPhone. In practice,
+every single undergraduate in North America whether they own an iPhone. In practice,
 directly computing population parameters is often time-consuming and costly, and sometimes impossible. 
 
 A more practical approach would be to collect measurements for a **sample**: a subset of
 individuals collected from the population. We can then compute a **sample estimate**&mdash;a numerical
-characteristic of the sample&mdash;that estimates the population parameter. For example, suppose we randomly selected 100 undergraduate students across North America (the sample) and computed the proportion of those
+characteristic of the sample&mdash;that estimates the population parameter. For example, suppose we randomly selected ten undergraduate students across North America (the sample) and computed the proportion of those
 students who own an iPhone (the sample estimate). In that case, we might suspect that that proportion is a reasonable estimate of the proportion of students who own an iPhone in the entire population. 
 
 ```{r 11-population-vs-sample, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Population versus sample", fig.retina = 2}
```

Comment:
github won't let me leave a comment further down, so i'll put it here

(around line 82 ish)

"And the list of population params we might want to calculate goes on"  -- awk sentence. could probably just remove.